src/test/java/uk/gov/justice/digital/hmpps/keyworker/config/LocalStackConfig.kt

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.digital.hmpps.keyworker.config
+
+import org.slf4j.LoggerFactory
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.testcontainers.containers.localstack.LocalStackContainer
+
+@Configuration
+@ConditionalOnExpression("'\${sqs.provider}'.equals('localstack') and '\${sqs.embedded}'.equals('true')")
+open class LocalStackConfig {
+  companion object {
+    private val log = LoggerFactory.getLogger(this::class.java)
+  }
+
+  @Bean
+  open fun localStackContainer(): LocalStackContainer {
+    log.info("Starting embedded localstack...")
+    val localStackContainer: LocalStackContainer = LocalStackContainer()
+        .withServices(LocalStackContainer.Service.SQS)
+        .withEnv("HOSTNAME_EXTERNAL", "localhost")
```

Comment:
what does this do? `HOSTNAME_EXTERNAL`

Reply:
no idea, @mikehalmamoj wrote it originally :grinning:

Reply:
It's mentioned here: https://github.com/localstack/localstack#configurations.  Apparently it default to localhost anyway :man_facepalming: .  I can't remember if I put it there for a reason or just copied and pasted?